web/src/containers/activity/EntryDetails.test.js

```diff
@@ -1,22 +1,37 @@
 import { shallow } from 'enzyme';
 import React from 'react';
 
-import { plain as EntryDetails, mapStateToProps } from './EntryDetails';
+import { removeActivity } from '../../actions/editActivity';
+
+import {
+  plain as EntryDetails,
+  mapStateToProps,
+  mapDispatchToProps
+} from './EntryDetails';
 
 describe('the (Activity) EntryDetails component', () => {
   const props = {
     activityIndex: 2,
     activityKey: 'activity key',
     fundingSource: 'money pit',
-    name: 'activity name'
+    name: 'activity name',
+    remove: jest.fn()
   };
 
+  beforeEach(() => {
+    props.remove.mockClear();
+  });
+
   test('renders correctly with the modal closed', () => {
     const component = shallow(<EntryDetails {...props} />);
     expect(component).toMatchSnapshot();
 
     const review = component.find('Review').dive();
-    review.find('Button').simulate('click');
+    // First button is the delete button; second is the edit button
+    review
+      .find('Button')
+      .at(1)
+      .simulate('click');
```

Comment:
Another separate test for button 0, to verify that the `props.remove` mock is called after it's clicked would be good.